app/core/config.py

```diff
@@ -1,7 +1,8 @@
 import os
-from typing import Final
+from typing import Final, Optional
 
 PROJECT_NAME = "navigator"
+DEVELOPMENT_MODE: bool = os.getenv("DEVELOPMENT_MODE", "False").lower() == "true"
```

Comment:
Is this variable on used for line 15-16 in this file? 
If so, wondering whether we just rely on people not having env vars set that they don't mean to and just default to None?

Reply:
@THOR300 updated